contents/articles/2017-08_pupil_mobile/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: Introducing Pupil Mobile
+date: Thu Aug 05 2017 14:36:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil Mobile is a companion app to Pupil Capture and Pupil Service. The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. Preview video and other sensor streams and stream video data over a WiFi..."
+featured_img: "../../../../media/images/blog/pupil_mobile.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/pupil_mobile.jpg"
+---
+
+We are excited to introduce Pupil Mobile! A companion app to Pupil Capture and Pupil Service, currently in public alpha at [Google Play Store](https://play.google.com/store/apps/details?id=com.pupillabs.pupilmobile)! 
+
+<div class="Feature-video-container-16by9">
+  <iframe class="Feature-video u-padTop--2" src="https://www.youtube.com/embed/atxUvyM0Sf8?rel=0" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+</div>
+<div class="small u-padBottom--2">Introducing Pupil Mobile</div>
+
+The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. You can preview video and other sensor streams on the Android device and stream video data over a WiFi network to other computers (clients) running Pupil Capture. And also save video from your Pupil eye tracking headset other sensor data locally on your Android device.
+
+Feel free to experiment with Pupil Mobile, but do not rely on the app to work all the time! Many features and environments are still untested. If you have trouble please open an [issue](https://github.com/pupil-labs/pupil-mobile-app/issues). The Pupil Labs development team will not be able to provide support via video or email for issues related to the Pupil Mobile Android App.
+
+Currently the application only supports Pupil latest release - [Pupil v0.9.13](https://pupil-labs.com/blog/2017-08/pupil-capture-player-and-service-release-v0-9-13/). Check out the blog post to read more about what's new in the release.
+
+For more information on Pupil Mobile, head over to [Pupil docs](https://docs.pupil-labs.com/#pupil-mobile).
```

Comment:
We're currently working on improving docs for Pupil Mobile, but a first draft of the docs is available at [docs.pupil-labs.com]((https://docs.pupil-labs.com/#pupil-mobile).

We look forward to seeing what you do with Pupil Mobile and to your feedback!